R-package/src/lightgbm_R.h

```diff
@@ -485,22 +491,26 @@ LIGHTGBM_C_EXPORT SEXP LGBM_BoosterCalcNumPredict_R(
 );
 
 /*!
-* \brief make prediction for a new data set
+* \brief make prediction for a new Dataset
 *        Note:  should pre-allocate memory for out_result,
 *               for normal and raw score: its length is equal to num_class * num_data
 *               for leaf index, its length is equal to num_class * num_data * num_iteration
-* \param handle handle
+* \param handle Booster handle
 * \param indptr pointer to row headers
 * \param indices findex
 * \param data fvalue
 * \param num_indptr number of cols in the matrix + 1
 * \param nelem number of non-zero elements in the matrix
 * \param num_row number of rows
-* \param is_rawscore
-* \param is_leafidx
+* \param is_rawscore 1 to get raw predictions, before transformations like
+*                    converting to probabilities, 0 otherwise
+* \param is_leafidx 1 to get record of which leaf in each tree
+*                   observations fell into, 0 otherwise
+* \param is_predcontrib 1 to get feature contributions, 0 otherwise
```

Comment:
Missed `start_iteration`.

Reply:
updated in https://github.com/microsoft/LightGBM/pull/4257/commits/9c173115582f1bb3793456beabbc1ace2b778899